_pages/releases.md

```diff
@@ -1,10 +1,5 @@
 ---
 title: Releases
-page_title: |
-  Releases
-  <a class="icon" href="/releases/feed.xml" rel="alternate" title="Newsfeed for Crystal releases" aria-label="Releases newsfeed" type="application/atom+xml">
-    {%- include icons/rss.svg %}
-  </a>
 layout: page-wide
```

Comment:
yeah, I mean change this page-wide with wide and use the link-actions, or if there's an important part missing in the layout, put the action links in a parent layout.

Reply:
Actually, the front matter version already works with this layout as well.